using_images/s2i_images/nodejs.adoc

```diff
@@ -31,6 +31,18 @@ https://github.com/openshift/sti-nodejs[4] of Node.js.
 
 == Images
 
+ifdef::openshift-online[]
+RHEL 7 images are available through Red Hat's subscription registry:
```

Comment:
s/Red Hat's subscription registry/the Red Hat Registry/

See https://access.redhat.com/containers/#/faq.